validphys2/src/validphys/photon/compute.py

```diff
@@ -276,13 +310,77 @@ def alpha_em_fixed_flavor(self, q, alpha_ref, qref, nf):
         alpha_em at NLO : float
             target value of a
         """
+        alpha_ref = couplings_ref[1]
         lmu = 2 * np.log(q / qref)
-        den = 1.0 + self.beta0[nf] * alpha_ref * lmu
+        den = 1.0 + self.betas_qed[nf][0] * alpha_ref * lmu
         alpha_LO = alpha_ref / den
-        alpha_NLO = alpha_LO * (1 - self.b1[nf] * alpha_LO * np.log(den))
-        return alpha_NLO
+        alpha_NLO = alpha_LO * (1 - self.betas_qed[nf][1] * alpha_LO * np.log(den))
+        return [couplings_ref[0], alpha_NLO]
 
-    def set_thresholds_alpha_em(self):
+    def couplings_fixed_flavor_exa(self, q, couplings_ref, qref, nf):
+        """
+        Compute numerically the running running for nf fixed.
+        The RGEs for the two couplings are coupled by the mixed terms,
+        so must be solved together starting from the same qref.
+
+        Parameters
+        ----------
+        q : float
+            target scale
+        alph_aem_ref : float
+            reference value of alpha_em
+        qref: float
+            reference scale
+        nf: int
+            number of flavors
+
+        Returns
+        -------
+        alpha_em at NLO : float
+            target value of a
+        """
+        u = 2 * np.log(q / qref)
+
+        # integration kernel
+        def rge(_t, alpha, beta_qcd_vec, beta_qcd_mix, beta_qed_vec, beta_qed_mix):
```

Comment:
Would be good to have some comment ere, for example with readily accessible where to check this expression.